src/architecture/unixfs.md

```diff
@@ -0,0 +1,611 @@
+---
+title: UnixFS
+description: >
+  UnixFS is a Protocol Buffers-based format for describing files, directories,
+  and symlinks as DAGs in IPFS.
+date: 2022-10-10
+maturity: reliable
+editors:
+  - name: David Dias
+    github: daviddias
+    affiliation:
+      name: Protocol Labs
+      url: https://protocol.ai/
+  - name: Jeromy Johnson
+    github: whyrusleeping
+    affiliation:
+      name: Protocol Labs
+      url: https://protocol.ai/
+  - name: Alex Potsides
+    github: achingbrain
+    affiliation:
+      name: Protocol Labs
+      url: https://protocol.ai/
+  - name: Peter Rabbitson
+    github: ribasushi
+    affiliation:
+      name: Protocol Labs
+      url: https://protocol.ai/
+  - name: Hugo Valtier
+    github: jorropo
+    affiliation:
+      name: Protocol Labs
+      url: https://protocol.ai/
+
+tags: ['architecture']
+order: 1
+---
+
+UnixFS is a [protocol-buffers][protobuf]-based format for describing files,
+directories and symlinks as Directed Acyclic Graphs (DAGs) in IPFS.
+
+## Nodes
+
+A :dfn[Node] is the smallest unit present in a graph, and it comes from graph
+theory. In UnixFS, there is a 1-to-1 mapping between nodes and blocks. Therefore,
+they are used interchangeably in this document.
+
+A node is addressed by a [CID]. In order to be able to read a node, its [CID] is
+required. A [CID] includes two important pieces of information:
+
+1. A [multicodec], simply known as a codec.
+2. A [multihash] used to specify the hashing algorithm, the hash parameters and
+   the hash digest.
+
+Thus, the block must be retrieved; that is, the bytes which ,when hashed using the
+hash function specified in the multihash, gives us the same multihash value back.
+
+In UnixFS, a node can be encoded using two different multicodecs, listed below. More details are provided in the following sections:
+
+- `raw` (`0x55`), which are single block :ref[Files].
```

Comment:
the partial ranges of data in multi-block files are also encoded as `raw` nodes, right?